docs/authoring.md

```diff
@@ -23,45 +25,77 @@ Accepted `<TargetFramework>` properties |
+
+``` nuspec
+  <files>
+    <file src="build\MyAuthoredComponent.CsWinRT.targets"                    target="build\MyAuthoredComponent.CsWinRT.targets" />
+    <file src="build\MyAuthoredComponent.targets"                            target="build\MyAuthoredComponent.targets" />
+    <file src="buildTransitive\MyAuthoredComponent.CsWinRT.targets"          target="buildTransitive\MyAuthoredComponent.CsWinRT.targets" />
+    <file src="buildTransitive\MyAuthoredComponent.targets"                  target="buildTransitive\MyAuthoredComponent.targets" />
+    <file src="lib\native\MyAuthoredComponent.dll"                           target="lib\native\MyAuthoredComponent.dll" />
+    <file src="lib\native\MyAuthoredComponent.winmd"                         target="lib\native\MyAuthoredComponent.winmd" />
+    <file src="lib\native\Microsoft.Windows.SDK.NET.dll"                     target="lib\native\Microsoft.Windows.SDK.NET.dll" />
+    <file src="lib\native\WinRT.Host.Shim.dll"                               target="lib\native\WinRT.Host.Shim.dll" />
+    <file src="lib\native\WinRT.Runtime.dll"                                 target="lib\native\WinRT.Runtime.dll" />
+    <file src="lib\net5.0-windows10.0.19041\MyAuthoredComponent.dll"         target="lib\net5.0-windows10.0.19041\MyAuthoredComponent.dll" />
+    <file src="lib\net5.0-windows10.0.19041.0\MyAuthoredComponent.dll"       target="lib\net5.0-windows10.0.19041.0\MyAuthoredComponent.dll" />
+    <file src="lib\net5.0-windows10.0.19041.0\MyAuthoredComponent.winmd"     target="lib\net5.0-windows10.0.19041.0\MyAuthoredComponent.winmd" />
+    <file src="lib\net5.0-windows10.0.19041.0\Microsoft.Windows.SDK.NET.dll" target="lib\net5.0-windows10.0.19041.0\Microsoft.Windows.SDK.NET.dll" />
+    <file src="lib\net5.0-windows10.0.19041.0\WinRT.Host.Shim.dll"           target="lib\net5.0-windows10.0.19041.0\WinRT.Host.Shim.dll" />
+    <file src="lib\net5.0-windows10.0.19041.0\WinRT.Runtime.dll"             target="lib\net5.0-windows10.0.19041.0\WinRT.Runtime.dll" />
+    <file src="runtimes\win-x64\native\WinRT.Host.dll"                       target="runtimes\win-x64\native\WinRT.Host.dll" />
+    <file src="runtimes\win-x86\native\WinRT.Host.dll" 			     target="runtimes\win-x86\native\WinRT.Host.dll" />
+  </files>
+```
 
+### For native app (C++) consumption
+You'll need to use [C++/WinRT](https://docs.microsoft.com/en-us/windows/uwp/cpp-and-winrt-apis/intro-to-using-cpp-with-winrt) to consume the `winmd` too, so make sure you have it installed, and have added `#include <winrt/MyAuthoredComponent.h>` to the file `pch.h` of the native app.  
 
-### Add the reference to your authored component
-**In Visual Studio** Under the Project node, right click on "References", click "Add Reference", then "Browse" and add the `.winmd` file generated in your authored component. 
+For C++/WinRT, you need to add a reference in the native app for the WinMD of your authored component. 

```

Comment:
WinMD/winmd - consistent casing

Reply:
is it alright to leave as "winmd" in the nuspec example, but in all prose use "WinMD" ?